app/scripts/metamask-controller.js

```diff
@@ -4789,10 +4789,7 @@ export default class MetamaskController extends EventEmitter {
     const filterMiddleware = createFilterMiddleware(proxyClient);
 
     // create subscription polyfill middleware
-    const subscriptionManager = createSubscriptionManager({
```

Comment:
do we not still want to update this to use `@metamask/eth-json-rpc-filters`? I saw that in your original work but wasn't sure if it was necessary or just a routine update to latest?

Reply:
fwir it started as a 'maybe this will fix it' and just stayed in as a general maintenance item. Might as well add it back in tho

Reply:
done!